src/Entity/Membership.php

```diff
@@ -148,9 +148,10 @@ public static function baseFieldDefinitions(EntityTypeInterface $entity_type) {
       ->setDescription(t('The Membership type/bundle.'))
       ->setSetting('target_type', 'membership_type')
       ->setRequired(TRUE);
+
     $fields['user_id'] = BaseFieldDefinition::create('entity_reference')
-      ->setLabel(t('Authored by'))
-      ->setDescription(t('The owner of the Membership entity.'))
+      ->setLabel(t('Primary user'))
```

Comment:
Ok, will change.

We do plan to support multiple users per membership, that is a requirement for several of our clients. So this will essentially be the author who can "change own" and add/remove users to a membership if more than one is allowed. We will want the API to support this, without requiring it...